git-branchless-hook/src/lib.rs

```diff
@@ -456,7 +456,12 @@ fn hook_reference_transaction(effects: &Effects, transaction_state: &str) -> eyr
                  ref_name,
                  old_oid: _,
                  new_oid: _,
-             }| !should_ignore_ref_updates(ref_name),
+             }| {
+                !should_ignore_ref_updates(ref_name)
```

Comment:
Rather than render this to a string, you'll want to write `match CategorizedReferenceName::new(...) { ... }` and check for the remote branch case.

Reply:
Ah I'm new to rust, is this the right way to match against enum variants?